lighthouse-core/lib/i18n/i18n.js

```diff
@@ -76,6 +76,8 @@ const UIStrings = {
   columnTransferSize: 'Transfer Size',
   /** Label for a column in a data table; entries will be the names of arbitrary objects, e.g. the name of a Javascript library, or the name of a user defined timing event. */
   columnName: 'Name',
+  /** Label for a column in data table; entries will be the duration, in milliseconds, of different timing measurements, e.g. the "Time to Interactive" or "First Contentful Paint". */
```

Comment:
can the description match the string a bit better? The "exceeded" context from the old description in `performance-budget.js` seems helpful here